### ISSUE TITLE
Accessibility audit - no title on iFrame

### ISSUE THREAD

**Issue by robertkruzel** [intbot/ng2-pdfjs-viewer#71]
As a user running aXe Accessibility audit tools on a page using ng2-pdfjs-viewer, I am notified of a violation (*** edit cuz of html tags *** frame or iframe must have a title) which is a violation of type: Section 508: MUST.

Looking at the source, perhaps something as simple as a static title would suffice?

**Comment by codehippie1** [comment]
@robertkruzel This is done. Please check latest npm package
